pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -1599,6 +1599,12 @@ public void openLedgerComplete(ManagedLedger ledger, Object ctx) {
                                         .exceptionally((ex) -> {
                                             log.warn("Replication or dedup check failed."
                                                     + " Removing topic from topics list {}, {}", topic, ex);
+                                            persistentTopic.getTransactionBuffer()
+                                                    .closeAsync()
+                                                    .exceptionally(t -> {
+                                                        log.error("[{}] Close transactionBuffer failed", topic, t);
+                                                        return null;
```

Comment:
Do you know if we need to return here?

Reply:
CompletableFuture#exceptionally() requires a `Function` arg
![image](https://user-images.githubusercontent.com/95597048/211513471-66e4d4bb-3b24-4ac5-9d8c-9866b4195144.png)
and `Function` requires a return value.
![image](https://user-images.githubusercontent.com/95597048/211513648-f07c6ec6-29ea-46bf-aa6b-18f26ca63ae4.png)